web/src/App.css

```diff
@@ -1,40 +1,95 @@
-.App {
+body {
```

Comment:
Can you set base font size for whole app?

```css
html {
  font-size: 12px;
}
```

Then everywhere, please use `rem` instead of `px` for fonts. In our case, `1rem` = `12px`. Please also use at most 1/4 multipliers (so `0.25rem`, `0.5rem`, `0.75rem`, `1.5rem`, `3rem` etc.)

Reply:
Done :)